app/helpers/rails_admin/application_helper.rb

```diff
@@ -26,11 +26,11 @@ def edit_user_link
       content = [
         RailsAdmin::Config.show_gravatar && _current_user.email.present? && image_tag("#{request.ssl? ? 'https://secure' : 'http://www'}.gravatar.com/avatar/#{Digest::MD5.hexdigest _current_user.email}?s=30", alt: ''),
         content_tag(:span, _current_user.email),
-      ].compact.join.html_safe
+      ].map(&:presence).compact.join.html_safe
       if (edit_action = RailsAdmin::Config::Actions.find(:edit, controller: controller, abstract_model: abstract_model, object: _current_user)).try(:authorized?)
         link_to content, rails_admin.url_for(action: edit_action.action_name, model_name: abstract_model.to_param, id: _current_user.id, controller: 'rails_admin/main'), class: 'nav-link'
       else
-        content_tag :span, content
+        content.sub('<span>', '<span class="nav-link">').html_safe
```

Comment:
```suggestion
        content_tag :span, content, class: 'nav-link'
```
I would suggest this, because it's simpler and achieves the same style.

Reply:
this would cause nested spans, but would also fix the render issue.